src/components/Icon/index.examples.js

```diff
@@ -64,6 +64,13 @@ storiesOf('Icons')
     <Icon type='lesson' />
   ))
 
+  .addWithInfo('Refresh', () => (
+    <Icon
+      type='refresh' 
+      spin
+    />
+  ))
```

Comment:
@trevordmiller right, I'm just of the opinion that it would be better to go to the icon you want by name instead of scrolling through the long combinations list to find the one you want :)